src/schema/text_options.rs

```diff
@@ -33,13 +34,25 @@ impl TextOptions {
         self.indexing = Some(indexing);
         self
     }
+
+    /// Returns true if the text is already tokenized in the form of TokenString
+    pub fn is_tokenized(&self) -> bool {
```

Comment:
This is not only spot, but I suggest we use the word `pretokenized` every where.

Reply:
You mean changing only TextOptions.is_tokenized -> TextOptions.is_pretokenized, TextOptions.set_tokenized -> TextOptions.set_pretokenized and Value.tokenized_text -> Value.pretokenized_text or also TokenizedString -> PretokenizedString, Value::TokStr -> Value::PretokStr ?

Reply:
Everyyyyywhere :)